### DISCUSSION TITLE
Official Docker Image

### DISCUSSION THREAD

**Discussion by trymeouteh** [nushell/nushell#13580] [Ideas]
Please consider releasing an official Nushell docker image. Similar to the Bash docker image but for Nushell. Will be useful when needing to test out scripts in a older version of Nushell.

**Comment by fdncred** [comment]
All it needs is a contributor who is passionate about creating and maintaining it.

**Comment by ghost** [comment]
there already is a unofficial one: `nixery.dev/shell/nushell`.  
but if anyone is interrested in maintaining a "official" one you can take [my version](https://github.com/Jan9103/dockerized_nushell) as base

**Comment by trymeouteh** [comment]
Looks like there is an official docker image now for Nushell

https://www.nushell.sh/book/installation.html#docker-container-images

I have not tested it out yet in Podman
